Formula/v8@5.8.rb

```diff
@@ -0,0 +1,118 @@
+# Track Chrome stable.
+# https://omahaproxy.appspot.com/
+class V8AT58 < Formula
+  desc "Google's JavaScript engine"
+  homepage "https://github.com/v8/v8/wiki"
+  url "https://chromium.googlesource.com/chromium/tools/depot_tools.git"
```

Comment:
You'll need to find a way to reuse the cache, and sync it up with the new v8 version as the version changes. It won't really be useful if it requires the user to blow it away manually. I assume gclient sync is able to do that without re-downloading everything if you pass it the current version.